charmcraft/commands/store/__init__.py

```diff
@@ -1105,6 +1105,17 @@ class FetchLibCommand(BaseCommand):
 
         The first time a library is downloaded the command will create the needed
         directories to place it, subsequent fetches will just update the local copy.
+
+        You can specify the library to update/download building its fully qualified
+        name with the charm and library names, and the desired API version. For
```

Comment:
This was a bit hard to parse the first time I read it. Maybe change to "...library to update or download by building its fully qualified name..."?

Reply:
thanks! improved